src/ajax.js

```diff
@@ -18,7 +18,8 @@ define( [
 var
 	r20 = /%20/g,
 	rhash = /#.*$/,
-	rts = /([?&])_=[^&]*/,
+	rantiCache = /(?:[?&])_=[^&]*&?/,
```

Comment:
Oh good catch, that was from an earlier failed attempt to solve the problem without having to make a second pass to remove the empty `?` querystring. I don't think its needed now.